src/_gettsim/functions_loader.py

```diff
@@ -571,6 +606,129 @@ def aggregation_func(source_col, group_id):
     return aggregation_func
 
 
+def _create_parent_child_link_functions(
+    user_and_internal_functions: dict[str, Callable],
+    data_cols: list[str],
+    user_provided_interpersonal_links_specs: dict[str, dict[str, str]],
+) -> dict[str, Callable]:
+    """
+    Create function dict with functions that link parent and child variables.
+    """
+
+    _, link_dict = load_aggregation_and_link_dict()
+
+    link_dict = {**link_dict, **user_provided_interpersonal_links_specs}
+
+    _fail_if_not_dict_of_dicts(link_dict)
+
+    link_functions = {
+        link_col: _create_one_link_func(
+            link_spec,
+            user_and_internal_functions,
+            data_cols,
+        )
+        for link_col, link_spec in link_dict.items()
+        if link_spec["source_col"] in user_and_internal_functions
+    }
+
+    return link_functions
+
+
+def _create_one_link_func(
```

Comment:
```suggestion
def _create_one_interpersonal_link_func(
```

Note: Multiple references to parents & children, which won't be caught by search and replace in the function body.